packages/helpers/qrcode-modal/src/browser/languages/fr.ts

```diff
@@ -1,8 +1,17 @@
-export default {
+import { TextMap } from "../types";
+
+const fr: TextMap = {
   choose_preferred_wallet: "Choisissez votre portefeuille préféré",
   connect_mobile_wallet: "Se connecter au portefeuille mobile",
   scan_qrcode_with_wallet: "Scannez le QR code avec un portefeuille compatible WalletConnect",
-  return_to_mobile_options: "Retour aux options du portefeuille mobile",
-  view_qrcode_option: "Voir le QR code",
   connect: "Se connecter",
+  // TODO: review following translations
```

Comment:
Could you take a look at the following translations? cc @PierrickGT

Reply:
@pedrouid looks good to me.